testsuite/module-publish/src/packages/complex/sources/liquid_pool/liquidity_pool.move

```diff
@@ -0,0 +1,612 @@
+/// Copied from aptos-move/move-examples/swap/sources/liquidity_pool.move
+///
+/// For volatile pairs, the price and reserves can be computed using the constant product formula k = x * y.
+/// For stable pairs, the price is computed using k = x^3 * y + x * y^3.
+module publisher_address::liquidity_pool {
```

Comment:
Unlike ambassador.move, did we have to copy the file here to make some custom changes?